crates/oxc_codegen/src/lib.rs

```diff
@@ -72,6 +76,7 @@ pub struct Codegen<'a, const MINIFY: bool> {
     prev_reg_exp_end: usize,
     need_space_before_dot: usize,
     print_next_indent_as_space: bool,
+    binary_expr_stack: Vec<BinaryExpressionVisitor<'a>>,
```

Comment:
TODO: try `smallvec`

Reply:
importing `smallvec` somehow caused other compile errors. Try `smallvec` in another PR.